crt_portal/cts_forms/migrations/0054_report_public_id.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 2.2.10 on 2020-03-09 22:44
+import random
+
+from django.db import migrations, models
+
+
+def add_salt():
+    # adding some non-ambiguous characters to salt the ids, this only needed for people asking abut their submission via phone, email or mail. There are no public automated lookups at this time. You could guess a range of IDs that might be assigned on a given day, but adding the letters adds 13,824 permutations to each of those records, so it would be labor intensive and noticeable to call in with questions if you were trying to guess at public_id that was not yours.
+    characters = ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'J', 'K', 'L', 'M', 'N', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z']
```

Comment:
Nice opportunity to use [string constants](https://docs.python.org/3/library/string.html#string-constants) from Python,

```python
import string
characters = string.ascii_uppercase
```